adage/analyze/models.py

```diff
@@ -218,3 +218,17 @@ def __unicode__(self):
         return "Model: %s, Node: %s, Gene: %s" % (self.node.mlmodel.title,
                                                   self.node.name,
                                                   self.gene.entrezid)
+
+
+class ExpressionValue(models.Model):
+    """
+    This class models the many-to-many relationship between Sample and Gene.
+    Each sample/gene pair has an expression value of floating type.
+    """
+    sample = models.ForeignKey(Sample, on_delete=models.PROTECT)
+    gene = models.ForeignKey(Gene, on_delete=models.PROTECT)
+    value = models.FloatField()
+
+    def __unicode__(self):
+        return "Sample %s and Gene %s with expression value %f" % (
```

Comment:
Hmm, I think I would phrase this more like "Expression value %f for Sample %s and Gene %s", so that the emphasis is on the fact that this is an Expression value. What do you think?